baseband_tasks/tests/test_dispersion.py

```diff
@@ -287,3 +288,46 @@ def test_disperse_negative_dm(self):
         # Lower sideband [1] is dedispersed to earlier.
         assert p[10, 0] > 0.99 and p[9, 0] < 0.006
         assert p[9, 1] > 0.99 and p[10, 1] < 0.006
+
+
+class TestDispersSample(TestDispersion):
+
+    @pytest.mark.parametrize('reference_frequency', REFERENCE_FREQUENCIES)
```

Comment:
These tests do not make that much sense for incoherent (de)dispersion, since the setup uses a complex signal with two channels with the same frequency, but different sideband. I think it makes more sense to set up tests that are more specific to incoherent (de)dispersion. Also, since samples are never changed, the comparison can be done more directly.